### ISSUE TITLE
Universal Pattern Component

### ISSUE THREAD

**Issue by thecodeclimber** [ramsayinnovations/ramsay-site#102]
Let's create a single, universal pattern component in the `develop--clone` branch.  This component would replace the following components:

- LeftCurveLines.jsx
- LeftPattern.jsx
- MobileRightCurveLines.jsx
- RightCurveLines.jsx
- RightPattern.jsx

All of these components have the same exact structure -- namely:

```
import PropTypes from "prop-types";

const COMPONENT_NAME = ({ classes }) => (
  <div className={classes.wrapper}>
    <img src="/img/IMAGE_NAME" className={classes.img} />
  </div>
);

COMPONENT_NAME.propTypes = {
  classes: PropTypes.shape({
    wrapper: PropTypes.string,
    img: PropTypes.string,
  }),
};

export default COMPONENT_NAME;
```

We could easily create one, single component which takes a `classes` and `img` prop (with the `img` prop receiving he name of the image file).

Please consult with @MaksymTaratuta55 if need be.

**Comment by kirti75waytech** [comment]
@MaksymTaratuta55 @thecodeclimber I have resolved this issue and created its PR #132. Please review it and let me know if any changes are required.
Thanks

**Comment by kirti75waytech** [comment]
@thecodeclimber I have resolved the conflicts for the PR #132. Please review it.
